topics/statistics/tutorials/intro_deep_learning/workflows/Intro_To_Deep_Learning.ga

```diff
@@ -68,7 +68,7 @@
                     "name": "y_test"
                 }
             ],
-            "label": "X_train",
+            "label": "Y_test",
```

Comment:
Hi Anup,
Since the test/train lables are vectors (not matrices), maybe we can use lowercase here? (y_test instead of Y_Test)

Reply:
Also, step 4's label on line 118 should revert back to null (right now its y_train. I did not realize steps start with 0, so my previous changes were off by 1)

Reply:
I'll apply the changes and push to my branch shortly. Could you please review the original PR (https://github.com/galaxyproject/training-material/pull/2288)? Thanks!

Reply:
Ok, my changes are pushed.